### PR TITLE
Added new values to allow custom ports

### PR DESCRIPTION
**Pull request by abelperezok**
This PR is related to the following issues: 

* https://github.com/open-telemetry/opentelemetry-helm-charts/issues/214
* https://github.com/open-telemetry/opentelemetry-operator/issues/896

It includes updates to the `opentelemetry-operator` chart to allow custom values to be used for ports:

```
--set manager.ports.metricsPort=8085
--set kubeRBACProxy.ports.proxyPort=8445
--set manager.ports.webhookPort=9445
--set manager.ports.healthzPort=8005
```

The default values for all these ports are the same as before and it's reflected in `values.yaml`

```yaml
manager:
# other values ...
  ports:
    metricsPort: 8080
    webhookPort: 9443
    healthzPort: 8081

kubeRBACProxy:
# other values ....
  ports:
    proxyPort: 8443
```

:warning: At this point, it's not going to work until PR https://github.com/open-telemetry/opentelemetry-operator/pull/899 is completed. If you want to test it meanwhile, please comment out or remove the lines: 

```yaml
            - --health-probe-addr=:{{ .Values.manager.ports.healthzPort }}
            - --webhook-port={{ .Values.manager.ports.webhookPort }}
```

https://github.com/abelperezok/opentelemetry-helm-charts/blob/new-values-custom-ports/charts/opentelemetry-operator/templates/deployment.yaml#L28-L29

Also, make sure you keep using the default values for both health check and webhook until the new version of the operator is published.

### CODE REVIEW

**Comment by TylerHelmuth** [comment]
@abelperezok can you change this to a draft PR until all other blocker PRs are merged?

**Comment by abelperezok** [comment]
@TylerHelmuth I've resolved the conflicts with the tag version, I kept the higher number.

**Review by dmitryax — APPROVED** [review]

**Comment by TylerHelmuth** [comment]
@abelperezok please bump the chart's patch version by 1.

**Comment by abelperezok** [comment]
@TylerHelmuth chart version is now 0.8.2

**Comment by TylerHelmuth** [comment]
@abelperezok chart testing failed twice in a row. Can you to
take a look?

**Comment by TylerHelmuth** [comment]
For comparison, here is a passing build of essentially main (I bumped the version so I could make a PR): https://github.com/TylerHelmuth/opentelemetry-helm-charts/pull/12

File: charts/opentelemetry-operator/values.yaml [hunk 1]
```diff
@@ -33,6 +37,8 @@ manager:
 kubeRBACProxy:
   image:
     repository: gcr.io/kubebuilder/kube-rbac-proxy
+  ports:
+    proxyPort: 8443
     tag: v0.11.0
```
Comment (TylerHelmuth) [inline charts/opentelemetry-operator/values.yaml:42]:
```suggestion
    tag: v0.11.0
  ports:
    proxyPort: 8443
```

I think this is the cause of the test failures.  The `tag` field needs to be under the `image` section not the `ports` section.  Thank goodness for automated tests.

Comment (abelperezok) [reply charts/opentelemetry-operator/values.yaml:42]: I could've been when I resolved the conflicts - merge human error ?

**Comment by TylerHelmuth** [comment]
@abelperezok thanks!

**Pull request merged**